src/tests/ftest/rebuild/io_conf_run.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019 Intel Corporation.
```

Comment:
Minor: Shall we move on to change this to 2020. Only if you need to submit the PR.